Example/Sentinel/AppDelegate.swift

```diff
@@ -39,10 +39,9 @@ private extension AppDelegate {
             tools: [
                 UserDefaultsTool(),
                 baseUrlTool,
-                CustomLocationTool(),
-//                BugsnatchTool(triggerActionConfig: EmailConfig()),
-//                LoggieTool(),
-//                AnalyticsCollectorTool(),
+                CustomLocationTool()
+//                CollarTool(),
```

Comment:
I think we can delete these comments

Reply:
We left them in the example project just for the sake of example on how you can add collar and analytics tools if you want to include them.